src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -1,5 +1,181 @@
 import React from 'react';
-import {View} from 'react-native';
+import {Image, View} from 'react-native';
+import PropTypes from 'prop-types';
+import Str from 'expensify-common/lib/str';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 
-const ValidationStep = () => <View />;
-export default ValidationStep;
+import {validateBankAccount} from '../../libs/actions/BankAccounts';
+
+import Button from '../../components/Button';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import TextInputWithLabel from '../../components/TextInputWithLabel';
+import Text from '../../components/Text';
+import BankAccount from '../../libs/models/BankAccount';
+import CONST from '../../CONST';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** Additional data for the account in setup */
+    achData: PropTypes.shape({
+
+        /** Bank account ID of the VBA that we are validating is required */
+        bankAccountID: PropTypes.number.isRequired,
+
+        /** State of bank account */
+        state: PropTypes.string,
+    }).isRequired,
+
+    /** Error message to display to user */
+    error: PropTypes.string,
+
+    /** Disable validation button if max attempts exceeded */
+    maxAttemptsReached: PropTypes.bool,
+};
+
+const defaultProps = {
+    error: '',
+    maxAttemptsReached: false,
+};
+
+class ValidationStep extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.submit = this.submit.bind(this);
+
+        this.state = {
+            amount1: '',
+            amount2: '',
+            amount3: '',
+            error: '',
+        };
+    }
+
+    submit() {
+        const amount1 = this.filterInput(this.state.amount1);
+        const amount2 = this.filterInput(this.state.amount2);
+        const amount3 = this.filterInput(this.state.amount3);
+
+        // If amounts are all non-zeros, submit amounts to API
+        if (amount1 && amount2 && amount3) {
+            const validateCode = [amount1, amount2, amount3].join(',');
+
+            // Make a call to bankAccounts
```

Comment:
Seems like maybe this thought wasn't finished. Should we say which command or that we are calling the API? Maybe don't really need the comment at all.